air/src/execution_step/errors/uncatchable_errors.rs

```diff
@@ -97,6 +98,10 @@ pub enum UncatchableError {
 
     #[error("failed to deserialize to CallServiceFailed: {0}")]
     MalformedCallServiceFailed(serde_json::Error),
+
+    /// Stream size estimate goes over a hardcoded limit.
+    #[error("stream size goes over the allowed limit of {STREAM_MAX_SIZE}")]
+    StreamSizeEstimateLimit,
```

Comment:
What does `estimate` means here?

Reply:
Technically usize can have wraparound.

Reply:
Let's not use this word, it could be get in another context, can we call it just 'StreamSizeLimitExceeded'

Reply:
Done